src/asmjs/asm_v_wasm.cpp

```diff
@@ -42,6 +42,7 @@ AsmType wasmToAsmType(Type type) {
     case f32: return ASM_FLOAT;
     case f64: return ASM_DOUBLE;
     case i64: return ASM_INT64;
+    case v128: assert(false && "v128 not implemented yet");
```

Comment:
for consistency with other asserts, the text should be in a comment on the same line. however, maybe we should change the others to be like this (later, if we decide so now). I don't feel strongly either way, but we should have a plan.